### PR TITLE
Added link in about us section and did changes in social media links

### PR DESCRIPTION
**Pull request by karanforprogram1**
Issue #122 solved

### CODE REVIEW

File: about.html [hunk 1]
```diff
@@ -273,7 +273,7 @@ <h5><b>Digital Inclusion Global Challenge</b></h5>
               <div class="col-md-3 col-sm-6 col-xs-12">
                   <div class="team-member">
                       <img class="img-circle our-awards" src="images/awards/uok.png" alt="biz minds award image">
```
Comment (pankajan05) [inline about.html:275]: Provide the `a href` tag to the images too. So the images must be clickable and it needs to redirect to the links.

File: about.html [hunk 2]
```diff
@@ -273,7 +273,7 @@ <h5><b>Digital Inclusion Global Challenge</b></h5>
...
-                      <h5><b>Biz Minds</b></h5>
+                      <h5><a href="https://www.kln.ac.lk/"><b>Biz Minds</b></a></h5>
```
Comment (pankajan05) [inline about.html:276]: It needs to open in new tab so use the `target="_blank"` inside the a href

**Review by pankajan05 — COMMENTED** [review]
Please do the changes that mentioned in the review

**Comment by karanforprogram1** [comment]
Hey @pankajan05 Added the needed changes

**Comment by karanforprogram1** [comment]
#105 solved

**Review by pankajan05 — APPROVED** [review]
Good job. The work is checked and validated.

**Pull request merged**